docs/environment-variables.md

```diff
@@ -0,0 +1,46 @@
+# Environment Variables
+
+This document outlines the set of environment variables that can be set to customize the behaviours at different levels.
```

Comment:
It needs to be clearly stated that these are unsupported and may be removed.

Reply:
Good point. Just added a note on this.